mojaloop/reporting-hub-bop-shell/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: This is the Shell application that accommodates BizOps microfrontends.
 name: reporting-hub-bop-shell
-version: 1.0.0
-appVersion: "2.1.1"
+version: 2.2.1
+appVersion: "2.2.1"
```

Comment:
App version and version don't need to match in this scenario. Because we may need changes to only chart without bumping up the docker image version.

Reply:
In this case we also update the docker image. What's the correct change in that case?

Reply:
Something like this
version: 1.1.0
appVersion: "2.2.1"